backend/views/syson-diagram-general-view/src/main/java/org/eclipse/syson/diagram/general/view/edges/AllocateEdgeDescriptionProvider.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.diagram.general.view.edges;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.sirius.components.view.builder.IViewDiagramElementFinder;
+import org.eclipse.sirius.components.view.builder.providers.IColorProvider;
+import org.eclipse.sirius.components.view.diagram.NodeDescription;
+import org.eclipse.syson.diagram.common.view.edges.AbstractAllocateEdgeDescriptionProvider;
+import org.eclipse.syson.diagram.general.view.GVDescriptionNameGenerator;
+import org.eclipse.syson.diagram.general.view.GeneralViewDiagramDescriptionProvider;
+import org.eclipse.syson.sysml.SysmlPackage;
+
+/**
+ * Used to create the Allocate edge description inside the General View diagram.
+ *
+ * @author Jerome Gout
+ */
+public class AllocateEdgeDescriptionProvider extends AbstractAllocateEdgeDescriptionProvider {
+
+    public AllocateEdgeDescriptionProvider(IColorProvider colorProvider) {
+        super(colorProvider);
+    }
+
+    @Override
+    protected String getName() {
+        return "GV Edge Allocate";
```

Comment:
I think it should be possible to use the GVDescriptionNameGenerator here, what do you think about?